modules/serverless-open-telemetry.adoc

```diff
@@ -96,7 +96,27 @@ spec:
     tracing:
       backend: "zipkin"
       zipkin-endpoint: "http://cluster-collector-collector-headless.tracing-system.svc:9411/api/v2/spans"
-      debug: "true"
+      debug: "true"  <- FIXME do we want consistency here?
+      sample-rate: "0.1" <1>
+----
+<1> The `sample-rate` defines sampling probability. Using `sample-rate: "0.1"` means that 1 in 10 traces are sampled.
+
+. Install Knative Eventing by creating the following `KnativeEventing` CR:
++
+.Example KnativeEventing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+    name: knative-eventing
+    namespace: knative-eventing
+spec:
+  config:
+    tracing:
+      backend: "zipkin"
+      zipkin-endpoint: "http://cluster-collector-collector-headless.tracing-system.svc:9411/api/v2/spans"
+      debug: "false"  <- FIXME do we want consistency here?
```

Comment:
Having `debug: "false"` here looks ok as long as we change the same stuff on line 99 as commented by @pierDipi